marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/mastering/common.mjs

```diff
@@ -18,7 +18,7 @@ function retrieveInterceptorFunction(interceptorObj, interceptorType) {
     return interceptorFunction;
 }
 
-function applyInterceptors(interceptorType, accumulated, interceptors, ...additionalArguments) {
+export function applyInterceptors(interceptorType, accumulated, interceptors, ...additionalArguments) {
```

Comment:
Shouldn't we keep the same format and do the export at the bottom with all the functions?